### PR TITLE
Display full name

### PR DESCRIPTION
**Pull request by jburel**
Re-add control to display the  full name  w/o having to expand the
left hand pane.

To test: 
 * Log in as any user
 * Click on the "Abc" button to display the full name
 * Check that the full name is displayed

This PR brings back a feature removed in gh-3770

cc @gusferguson @mporter-gre

### CODE REVIEW

**Comment by gusferguson** [comment]
@jburel 

Tested using OMERO.insight-5.1.3-69-3fabcc8-ice34-b590-mac_Java7+ user-3 read-only-1

Behaves as expected.
Good to merge.

**Comment by jburel** [comment]
Discussed with @gusferguson: screenshots will have to be updated but that can be done later on. (non critical)

**Comment by gusferguson** [comment]
@jburel - I will have them ready for 5.1.4 release - pretty straight forward.

**Comment by jburel** [comment]
@gusferguson: Thanks

File: components/insight/SRC/org/openmicroscopy/shoola/agents/treeviewer/browser/BrowserComponent.java [hunk 1]
```diff
@@ -1104,8 +1104,9 @@ public void displaysImagesName()
 		 if (model.getState() == DISCARDED)
 			 throw new IllegalStateException("This method cannot be invoked "+
 	                "in the DISCARDED state.");
+		 System.err.println(view.isPartialName());
```
Comment (joshmoore) [inline components/insight/SRC/org/openmicroscopy/shoola/agents/treeviewer/browser/BrowserComponent.java]: `System.err` heads up

**Comment by jburel** [comment]
@joshmoore: Thanks good opportunity to try the online edit !!!

**Comment by jburel** [comment]
--rebased-to #4102

**Pull request merged**